### DISCUSSION TITLE
Does config subtititue quarkus. properties with QUARKUS_ envs?

### DISCUSSION THREAD

**Discussion by agreedSkiing** [quarkusio/quarkus#38809] [Q&A] [answered]
If I have `quarkus.datasource.blocking.jdbc.url = ${quarkus.datasource.jdbc.url:jdbc:postgresql://localhost:5433/quarkus}` will it then use the variabel export QUARKUS_DATASOURCE_JDBC_URL when running?

https://quarkus.io/guides/config-reference#with-environment-variables

**Comment by cescoffier** [comment]
\CC @radcortez

**Comment by radcortez** [comment, marked as answer]
> If I have `quarkus.datasource.blocking.jdbc.url = ${quarkus.datasource.jdbc.url:jdbc:postgresql://localhost:5433/quarkus}` will it then use the variabel export QUARKUS_DATASOURCE_JDBC_URL when running?
> 
> https://quarkus.io/guides/config-reference#with-environment-variables

Yes. Even for substitutions, the lookup rule still applies.
